Initial docs (#66)
EventStore/EventStore-Client-Dotnet: docs/docs/connecting-to-eventstore/setting-up-a-connection-with-net-dependency-injection.md

# Setting up a connection with .NET dependency injection

:::tip
Packages needed:
- `EventStore.Client.Grpc.Streams`   
- `EventStore.Client.Grpc.ProjectionManagement`
- `EventStore.Client.Grpc.Operations`
- `EventStore.Client.Grpc.UserManagement`
:::

> As part of the client packages provide `IServiceCollection` extension methods to make it easy to setup dependency injection in .NET Core web applications. 

Reviewer: ```suggestion
The client packages provide `IServiceCollection` extension methods that make it easy to setup dependency injection in .NET Core web applications. 
```
